README.md

```diff
@@ -44,8 +44,8 @@ First, prepare a samplesheet with your input data that looks as follows:
 `samplesheet.csv`:
 
 ```csv
-sample,lane,group,fastq_1,fastq_2,rundir
-CONTROL_REP1,1,GROUP1,AEG588A1_S1_L002_R1_001.fastq.gz,AEG588A1_S1_L002_R2_001.fastq.gz,200624_A00834_0183_BHMTFYDRXX
+sample,fastq_1,fastq_2,rundir,tags
+CONTROL_REP1,AEG588A1_S1_L002_R1_001.fastq.gz,AEG588A1_S1_L002_R2_001.fastq.gz,200624_A00834_0183_BHMTFYDRXX,"patient1"
```

Comment:
I prefer avoiding `patient` in the examples, since it suggests an application in a clinical setting and this pipeline is evidently no approved in-vitro diagnostic product or medical device.